aws/cloudformation/lambdas/sql-user/cfn-response.js

```diff
@@ -4,17 +4,24 @@
 exports.SUCCESS = "SUCCESS";
 exports.FAILED = "FAILED";
 
-exports.send = function(event, context, responseStatus, responseData, physicalResourceId, noEcho) {
-
+exports.send = function (
+  event,
+  context,
+  responseStatus,
+  responseData,
+  physicalResourceId,
+  noEcho
+) {
```

Comment:
our pre-commit hook linter `run_eslint_apps` in "hooks.rb" is not happy with these files, so I ran it with the `--fix` option to auto-fix these files.